geth/node/rpc_test.go

```diff
@@ -11,6 +13,17 @@ import (
 	"github.com/stretchr/testify/suite"
 )
 
+//==================================================================================================
```

Comment:
I think we should stop using separators, I don't know even a single case where they'd increase readability. Their usage often implies that it's time to separate entities or functionality over several files.